Packages/Sites/Neos.NeosIo/Resources/Private/Templates/NodeTypes/Reference.List.html

```diff
@@ -1,3 +1,25 @@
-<div {attributes -> f:format.raw()}>
-	{references -> f:format.raw()}
+{namespace io=Neos\NeosIo\ViewHelpers}<div {attributes -> f:format.raw()}>
+	<io:simpleForm actionUri="#reference-filter" class="form form--inline" method="get" id="reference-filter">
+		<fieldset>
+			<legend>Filter</legend>
+			<div class="form__item">
+				<label for="referenceType">Type</label>
+				<f:form.select id="referenceType" class="textInput" name="referenceType" options="{referenceTypes}" value="{referenceType}" optionLabelField="label" optionValueField="nodeData.identifier" prependOptionLabel="- all -" additionalAttributes="{onchange: 'this.form.submit()'}" />
+			</div>
+			<div class="form__item">
+				<input class="textInput" type="text" name="search" placeholder="Search term" value="{searchTerm}" onchange="this.form.submit()" />
+			</div>
+			<div class="form__item">
+				<label for="order">Sort by</label>
+				<f:form.select id="order" class="textInput" name="order" options="{datePublished: 'Publish date', launchDate: 'Launch date', projectVolume: 'Project volume'}" value="{order}" additionalAttributes="{onchange: 'this.form.submit()'}" />
+			</div>
+			<div class="form__item">
+				<label for="limit">Number of results</label>
+				<f:form.select id="limit" class="textInput" name="limit" options="{20: '20', 50: '50', 100: '100'}" value="{limit}" additionalAttributes="{onchange: 'this.form.submit()'}" />
```

Comment:
Does it make sense to have the limit selectable, a good default would be better I think.
How much results are to be expected per type?